source/heroes/ScreensaverHero.hx

```diff
@@ -0,0 +1,71 @@
+package heroes;
+
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.FlxG;
+
+class ScreensaverHero extends FlxSprite {
+    var HeroHeight:Int = 50;
+    var HeroWidth:Int = 25;
+    var StartingHeroColor:FlxColor = 0xFFFFFFFF;
+    var Blue:FlxColor = 0xFF0000FF;
+    var Yellow:FlxColor = 0xFFFFFF00;
+    var Green:FlxColor = 0xFF008000;
+    var Red:FlxColor = 0xFFFF0000;
+    var SpeedX:Float = FlxG.random.float(-50, 50);
+    var SpeedY:Float = FlxG.random.float(-50, 50);
+
+    public function new(X:Float=0, Y:Float=0)
+    {
+        super(X, Y);
+        super.makeGraphic(HeroWidth, HeroHeight, StartingHeroColor);
+        this.velocity.x = SpeedX;
+        this.velocity.y = SpeedY;
+    }
+    override public function update(elapsed:Float)
+    {
+        if (this.velocity.y > 0 && this.y >= FlxG.height){
+            this.y = 0; 
+        }
+        else if (this.velocity.y < 0 && (this.y + this.height) <= 0){
+            this.y = FlxG.height;
+        }
+
+        if (this.velocity.x > 0 && this.x >= FlxG.width){
+            this.x = -(this.width);   
+        }
+        else if (this.velocity.x < 0 && (this.x + this.width) <= 0){
+            this.x = FlxG.width; 
+        }
+        // Accelerate screensaver heroes if spacebar is pressed
+        if(FlxG.keys.pressed.SPACE)
+        {
+            SpeedX *= 1.01;
+            SpeedY *= 1.01;
+        }
+        // Reinitialize speeds if R is pressed
+        if(FlxG.keys.pressed.R)
```

Comment:
Note, if you like the rectangles wiggling in terror when you hold down `R`, no change needed. 

However, if you just want this to trigger when `R` is first pressed, you can use `FlxG.keys.justPressed.R` instead

Reply:
The wiggling in terror is a feature not a bug!